pages/dkp/kommander/2.2/projects/applications/platform-applications/index.md

```diff
@@ -12,6 +12,8 @@ The following table describes the list of applications that can be deployed to a
 Review the [project application service resource requirements](./platform-application-requirements/) to ensure that the attached clusters have sufficient resources.
 -->
 
+When deploying and upgrading applications, platform applications come as a bundle; they are tested as a single unit and you must deploy or upgrade them in a single process, for each workspace. This means all clusters in a workspace have the same set and version of platform applications deployed.
```

Comment:
```suggestion
When deploying and upgrading applications, platform applications come as a bundle; they are tested as a single unit and you must deploy or upgrade them in a single process, for each workspace. This means all clusters in a workspace have the same set and versions of platform applications deployed.
```

Reply:
I prefer 'set' to 'bundle'.  We have an 'airgapped bundle' which has all the parts we use to install/upgrade both Konvoy and Kommander (and OS dependencies) so I wonder if 'bundle' could be confusing here.